Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts

```diff
@@ -147,19 +161,7 @@ async function DeployUsingKuduDeploy(webDeployPkg, azureWebAppDetails, publishin
     }
     catch(error) {
         tl.error(tl.loc('Failedtodeploywebsite'));
-        isDeploymentSuccess = false;
-        deploymentError = error;
-    }
-
-    try {
-        tl._writeLine(await azureRESTUtility.updateDeploymentStatus(publishingProfile, isDeploymentSuccess));
-    }
-    catch(error) {
-        tl.warning(error);
-    }
-    
-    if(!isDeploymentSuccess) {
-        throw Error(deploymentError);
+        throw Error(error);
```

Comment:
You need to increase task patch version as well for both this task and machine group iis task